internal/command/migrate_to_v2/pg.go

```diff
@@ -77,10 +77,9 @@ func (m *v2PlatformMigrator) updateNomadPostgresImage(ctx context.Context) error
 }
 
 func (m *v2PlatformMigrator) migratePgVolumes(ctx context.Context) error {
-	app := m.appFull
 	regionsToVols := map[string][]api.Volume{}
 	// Find all volumes
-	for _, vol := range app.Volumes.Nodes {
+	for _, vol := range m.oldAttachedVolumes {
```

Comment:
note that this *looks* like it changes behavior, but it filters by whether there's an attachedAllocation below anyway, so this shouldn't change anything